src/autopas/containers/verletListsCellBased/verletListsCells/neighborLists/VLCAllCellsGeneratorFunctor.h

```diff
@@ -56,17 +70,58 @@ class VLCAllCellsGeneratorFunctor : public Functor<Particle, VLCAllCellsGenerato
     if (i.isDummy() or j.isDummy()) {
       return;
     }
-    auto dist = i.getR() - j.getR();
-    double distsquare = utils::ArrayMath::dot(dist, dist);
-    if (distsquare < _cutoffskinsquared) {
+    const auto dist = i.getR() - j.getR();
+    const double distSquare = utils::ArrayMath::dot(dist, dist);
+    if (distSquare < _cutoffSkinSquared) {
       // this is thread safe, only if particle i is accessed by only one
       // thread at a time. which is ensured, as particle i resides in a
       // specific cell and each cell is only accessed by one thread at a time
       // (ensured by traversals)
       // also the list is not allowed to be resized!
 
-      auto &[cellIndex, particleIndex] = _particleToCellMap[&i];
-      _neighborLists[cellIndex][particleIndex].second.push_back(&j);
+      // Depending on the targeted traversal neighbor lists are built differently
+      if constexpr (TraversalOptionEnum == TraversalOption::Value::vlc_c08) {
```

Comment:
Technically this is not needed anymore since we don't use the functor for building the `vlc_c08` lists any longer. The only purpose this serves is for comparative reasons. I think it should be kicked out when tackling #878.